lib/wanda_web/router.ex

```diff
@@ -12,6 +12,7 @@ defmodule WandaWeb.Router do
     pipe_through :api
 
     resources "/executions", ExecutionController, only: [:index, :show]
+    get "/groups/:id/executions/last", ExecutionController, :last
```

Comment:
I guess this removes the need to have the `group_id` filter in the `/executions` endpoint, as we would maybe better have `/groups/:id/executions`.
Anyway, let's not bother now. Both would do the same and 1st is already implemented

Reply:
the group_id filter could be still  helpfpul when we'll do the historical result view